### DISCUSSION TITLE
Project Name Selection

### DISCUSSION THREAD

**Discussion by niyaziulke** [bounswe/2021SpringGroup9#91] [General]
It's time to find a good name for our project. Here we can share and discuss our ideas.

**Comment by niyaziulke** [comment]
What about Postory? It's like "post" and "story", also sounds similar to "history".  "Pos" at the beginning sounds like an abbreviation of "position"  😺

**Comment by zcanfes** [comment]
Here are some more ideas: 

* Memorys
* Rememorie
* Storiemd
* Memoryte
* Memoryfell
* StoryKey
* Locationcast
* MemoryVault
* Itstory
* InMemory
* Storyaid

**Reply by zcanfes** [reply]
* MemoryMap

**Reply by zcanfes** [reply]
* MemoryAisle
